assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -81,9 +82,11 @@ function Panel({
     return persisted?.isCollapsed ?? collapsedByDefault;
   });
   const [expandToHeight, setExpandToHeight] = useState(
-    persisted?.expandToHeight || initialHeight
+    Math.min(persisted?.expandToHeight, maxHeight) || initialHeight
+  );
+  const [height, setHeight] = useState(
+    Math.min(persisted?.height, maxHeight) || initialHeight
```

Comment:
nit: if both `expandToHeight` and `height` are getting the same treatment, do you think there's any value in having it be shared, like `const panelHeight = Math.min(persisted?.height, maxHeight) || initialHeight;` and then just `useState(panelHeight)`. 

🤷  they're right next to each other and this is easy to read as is.

Reply:
I don't think we can combine them because some panels aren't open so we need to check both `persisted.height` and `persisted.expandedToHeight`

If the panel is closed, then height will be 0 or something, but we'd still need to know how big the panel needs to be expanded to.

Reply:
yes :) i misread what's being persisted in each. 🙃